sprint-8

zalivochkin opened the pull request.

zalivochkin committed:
sprint-8

ivannizh commented:
Привет!
Все запустил и пытаюсь скачать отчет, но он просто зависает и ничего не отдает.
![image](https://github.com/user-attachments/assets/96fa5a21-3259-4a32-a2ed-effbdd7f46b9)

zalivochkin committed:
fix keycloak url

zalivochkin committed:
Keycloak URL /protocol/openid-connect/token/introspect работает только с тем-же хостом, с которого токен был получен.
Чтобы проверять токен с хоста api_service надо добавить заголовок 'Host: localhost:8080'.

zalivochkin commented:
Привет!
Доотладил.
![api_reports](https://github.com/user-attachments/assets/ece1842b-f9c8-4bd5-b569-9a8cd5b2bafc)
